lib/screens/sign_in.dart

```diff
@@ -15,17 +15,21 @@ class SignInScreen extends StatefulWidget {
 
 class _SignInScreenState extends State<SignInScreen> {
   final _formKey = GlobalKey<FormState>();
+
+  Navigation _navigation;
```

Comment:
hi @nickmeinhold, are you wanting to use the navigation as a variable or did you forget to delete it?
im easy either way, thanks for the good work!

Reply:
Totally forgot, awesome spot, thanks!  Fixing now...

Reply:
Fixed.